content/kubermatic/main/architecture/concept/kkp-concepts/applications/default-applications-catalog/kube-vip/_index.en.md

```diff
@@ -0,0 +1,29 @@
++++
+title = "Kube-VIP Application"
+linkTitle = "Kube-VIP"
+enterprise = true
+date = 2024-01-16T12:57:00+02:00
+weight = 6
+
++++
+
+# What is Kube-VIP?
+
+For more information on the Kube-VIP, please refer to the [official documentation](https://kube-vip.io/)
```

Comment:
Can we add a short sentence here what Kube-VIP is?

Reply:
Oops missed it. nice catch! :) Adding it now.